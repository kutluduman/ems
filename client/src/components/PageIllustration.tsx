import { styled } from "@mui/material/styles";

const ImageWithPadding = styled("img")`
  padding: 40px;
`;

interface StyledIllustrationProps {
  src: string;
  alt: string;
}

const StyledIllustration = ({
  src,
  alt,
}: StyledIllustrationProps) => {
  return <ImageWithPadding src={src} alt={alt} />;
}
export default StyledIllustration;