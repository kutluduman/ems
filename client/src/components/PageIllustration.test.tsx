import { create } from "react-test-renderer";

import PageIllustration from "./PageIllustration";

describe("PageIllustration Component", () => {
  test("renders the PageIllustration Component", () => {
    const tree = create(<PageIllustration src="some source" alt="some alt" />);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});