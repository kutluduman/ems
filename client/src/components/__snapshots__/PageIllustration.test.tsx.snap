// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageIllustration Component renders the PageIllustration Component 1`] = `
<img
  alt="some alt"
  className="css-14z6jlm"
  src="some source"
/>
`;
